databases_201.rst

```diff
@@ -25,6 +25,84 @@ CouchDB
 Hadoop
 ------
 
+ElasticSearch
+-------------
+
+**Intro**
+
+*ElasticSearch* main logo - "you know, for search". It's near real-time search engine and document-oriented datastore. It's written in Java, so lots of processes depends on JVM (Java Virtual Machine). System has awesome REST API, its documents and queries - all is represented via JSON (https://en.wikipedia.org/wiki/JSON). All configuration can be set up in elasticsearch.yml file. Analogues: Apache Solr, Sphinx.
```

Comment:
The link to Wikipedia can be done via an External Hyperlink - see references in the other parts of the docs as well as the original: http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets

Another option would be to create a Glossary entry for JSON - I'll leave it up to you to decide.